packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/description.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+
+type TDescription = {
+    id: string;
+    description: string;
+};
+
+const Description = ({ id, description }: TDescription) => (
+    <React.Fragment>
+        {id === 'type-strategy' && (
+            <div key='description' className='quick-strategy__text'>
+                <Text text='sm' weight='normal'>
+                    {description}
```

Comment:
do we need localize here ?

Reply:
@rupato-deriv it has localize in the config file where it is from comes, we don’t need localize it twice